src/App.tsx

```diff
@@ -1,11 +1,11 @@
 import React, { Fragment } from "react";
 import { connect } from "react-redux";
-import { BrowserRouter as Router } from "react-router-dom";
+import { HashRouter as Router } from "react-router-dom";
```

Comment:
Part 2 - React app should use Hash router where all routing info is after the `#` in the url.

Reply:
We'll need to create some switch here between react/browser mode.

Reply:
@wbreza would some of Jordan's work be useful here?

Reply:
Yes, definitely - just didn't want to include it as part of this PR.